### ISSUE TITLE
Does not work if code contains ES6 syntax

### ISSUE THREAD

**Issue by marwanhilmi** [benbria/aliasify#43]
The aliasify transform will fail if code contains ES6 syntax. So far, both usage of `const` and template strings cause issues. I will follow up with exact error message.

**Comment by gunderson** [comment]
Also fails on `export MyExport from MyPackage` and fat-arrow functions

**Comment by gunderson** [comment]
also fails when using pkgify. I think this is a problem with browserify-transform-tools

**Comment by jwalton** [comment]
Yeah, likely this can be fixed just by upgrading falafel in browserify-transform-tools.  If someone wants to make a PR and try it out, I'm happy to merge it.

**Comment by gunderson** [comment]
Yep, I tracked it down to an old verson of acron in falafel. Went in and manually updated to acorn@3.2 and all is well.

**Comment by jwalton** [comment]
Um, yeah, I guess so.  -_-  At least I own browserify-transform-tools, so if node-falafel is fixed, it's easy to fix the others.

Although, this doesn't quite fix this perfectly, because browserify-transform-tools#makeRequireTransform is going to transform all the `require`s in your code, but I doubt it will nicely fix the `import`s.

As a workaround, you can add a Babel transform before you get to aliasify (so there won't be any ES6 code in what aliasify is transforming) but this only helps if you want something like Babel in your transform stack in the first place.

**Comment by gunderson** [comment]
I'm already running babelify along with browserify (and aliasify + stringify), but for some reason the parse through BTT didn't seem to be helped by putting the babel step first in the transform chain  ¯\_(ツ)_/¯

I'll just go ahead and go to falafel since that's the quickest fix.

**Comment by mmusket** [comment]
Are there any updates regarding this issue? 
I am running into the same problem.

**Comment by gunderson** [comment]
Depends on an upstream update of benbria/browserify-transform-tools#24

For now, you can manually bump the versions in node_modules.

**Comment by eightyfive** [comment]
Hi @gunderson. How do you "bump the versions in node_modules" manually? Could you explain further the process? I don't get how to tell browserify to use `falafel@2.0.0`.

Thanks.

**Comment by mmusket** [comment]
@eightyfive Go into the node_modules folder and find the package.json file inside the alaisfy folder. 
There you can change the old falafel dependency and update it. 
Obviously this is an untested change and might break things? and also if you upgrade the package the manual override will be lost.

**Comment by eightyfive** [comment]
Hi, thx.
What I did is change the version manually in:
```
node_modules/browserify-transform-tools/package.json
```
I also did install `falafel@2.0.0`.

Do I need to clear some npm cache so `browserify-transform-tools` grabs the v2 of `falafel`?

Because I still get the same error:
```
SyntaxError: Unexpected token (2:7) (while aliasify was processing...
```

Thanks for your help.

**Comment by jwalton** [comment]
Should be fixed in v2.1.0.
